### PR TITLE
Gives clockwork marauders a reasonable cost/cooldown

### PR DESCRIPTION
**Pull request by Robustin**
:cl: Robustin
balance: Clockwork marauders now take more time (+3s) and power (7x) to create.
balance: The "recent marauder" time limit is now 60 seconds, up from 20. The limit now has a significantly smaller effect on summon time but will act as a further cap on marauder summoning until it has passed.
tweak: The marauder cap will now only account for living cultists. 
/:cl:

I told Xhuis to fix this shit months ago. 

A marauder is summoned in 5 seconds using less power than a single fucking brass wall
**A single stargazer generates enough power for a marauder in 50 seconds, 38 seconds during warcult**
Someone actually thinks this is OK to cycle an endless stream of constructs from nothing

### CODE REVIEW

**Comment by optimumtact** [comment]
Please consider this a friendly warning, your PR's must have a properly descriptive title and body content that explains the purpose of the PR.

Your current title and or body content does not meet requirements, so please update it.

**Comment by Xhuis** [comment]
Buffing the power cost to obscene amounts will make it so that spamming it cripples the entire cult because people aren't attentive and will spam them anyways. Stargazers generate enough power but this is assuming people actually build enough well-hidden stargazers to fuel a massive amount.

I'd rather you just remove marauders outright instead of making them cost massively more power than any other scripture in the game because it's highly inconsistent. There are also far more elegant solutions. like just adding a flat cooldown on each new marauder.

Forgive me for not "fixing it months ago", I have a right to think on things without making kneejerk balance PRs, because that's how shadowling and old CC were both ruined. As far as I know, you don't control me, and I don't have to listen to anything you "tell me to fix."

**Comment by Robustin** [comment]
Some things are so obviously in need of a fix there's really no excuse for not. Marauders costing less power than a single fucking clockwork wall was a glaring and obvious issue.

**Comment by Xhuis** [comment]
In vacuum, that sounds silly. It's intentional, so that building defenses costs a hefty amount of power but scriptures generally don't, somewhat by necessity because if scripture cost a lot there's no way for it to be used in any reasonable way.

**Comment by AdamElTablawy** [comment]
Consider just making them constantly drain power as long as they're alive, and have a cooldown on summoning so they can't just constantly be chain shat out the moment one dies.

**Comment by optimumtact** [comment]
If your';e going to be fixing clockcult annoyances please TURN THE FUCKING SOUNDS DOWN ON SUMMON CHRIST

**Comment by Robustin** [comment]
I'd bring down some if there is evidence of ANYTHING else being a significant power drain.

As it stands you dump gazers into atmos tanks or edges of the AI sat, solars, abandoned rooms, whatever, then coast to a massive power glut because even a 3x3 of gazers can power an entire cult.

**Comment by AdamElTablawy** [comment]
Brass is usually a huge power drain, since cogscarabs usually put out a ton of it.

That being said, robustin, most gazer nests will make more than 10kw in like 10 seconds if the cult is competent, and thus it doesn't solve the spam issue, because a cult competent enough to actually cause problems with marauder spam while not crippling its own defenses and thus making the attack trivially easy won't have problems with the increased power cost.

Either put a cooldown on creating another one that goes into effect when one marauder dies, or give them a constant power draw as long as they're alive, or both.

**Comment by Robustin** [comment]
Im fine with a cooldown but I'm not wild about this idea that because the power system is so shitty we should give up any hope of balancing around it.

I warned Xhuis about gazers, I told him they would never promote good gameplay. And here we are.

**Comment by ChangelingRain** [comment]
Linking the power system to scripture was pretty much never a good idea, since it ended up making scripture cost *nothing* rather than restricting it.
Plus like, the sigil of transmission is still a thing for some reason so the cult's power is also soft-linked to station power, meaning it's possible for them to totally eliminate power requirements since the station's power amounts are *so out of wack*; each APC is 2500, each SMES is ~50000(200000 if you deconstruct it before draining), and that's a gap that's impossible to bridge.

**Comment by AdamElTablawy** [comment]
wait hold on

does the sigil of transmission actually drain power from the station @ChangelingRain

**Comment by Robustin** [comment]
It definitely used to, thought Xhuis's version just turned it into a pretty lame battery mechanic but since clock cultists are never tight on power I can't tell. 

It wouldn't be hard to make a decent power systems but gazers and transmission-style draining would have to go. I think integration cogs are a fair balance but ironically despite taking more time to place and being far more limited, they're only about 5% as effective as a single goddamn gazer.

**Comment by Xhuis** [comment]
Transmission sigils don't drain power anymore, no; integration cogs are in a bit of a strange spot. originally I planned to do more with the power system (i.e. have different generators with pros/cons that could be used interchangeably depending on strategy) but I ran dry on ideas and went with stargazers being the meta, which put it into the functional state of surplus/shortage. i'm still thinking on how to change the power system but i'm sick and have no coherent ideas atm

**Comment by Robustin** [comment]
You could do worse than just removing gazers and buffing cogs. It would also tamp down the ridiculous endless sabotage if the cult had to spend more time trying to cog APC's. 

At the very least have gazers ramp up over time so people have an incentive to be discrete instead of just pooping them out next to every window they see because they recoup costs in like 2 seconds.

**Comment by Robustin** [comment]
I updated it to 7,000 but I still think with the warcult discount its going to be trivial to maintain marauder spam under the current mechanics. Right now the cults only have 1-2 even paying attention to power generation and they're maintaining high power effortlessly. If, by some miracle, this PR did choke the clock cult on power levels, they have plenty of ways to expand generation besides "I dumped a square of gazers on a corner of the AI sat so we're pretty much good to go".

**Comment by KorPhaeron** [comment]
Power generation is just weird in general (as you said, it mostly devolves into spamming stargazers in the AI sat or in atmos tanks). I wonder how the mode would work with a lump sum of power at the beginning based on pop and any other resources you require you have to steal.

**Comment by KorPhaeron** [comment]
@robustin Agree with the premise of this PR but doubt this will even make a dent in the problem. Sure you want to burn gbp on this being merged?

**Comment by Robustin** [comment]
The only way we're going to reach some kind of sanity of CC power balance (and of course, marauder spam) is by adjusting costs and generation. 

I'm not ready to declare war on stargazers (and I'm still amused by the counterplay thats evolving around them) so it makes sense to focus on the totally irrational costs. This is such a simple and non-commital change I'm 100% behind trying it out. If we rationalize power generation down the road and start to reach a more coherent power economy for the CC then I'd be more than happy to see this cost get brought down. My only pause is to whether I should try to throw in a global cooldown for marauders too so that one person running the usual "sit in perma spamming marauders" routine doesn't drain the entire CC against their wishes.

**Comment by Robustin** [comment]
Ok the changes will need a quick review but I utilized Xhuis's previous attempt at discouraging spam to add a harder limit on how often you can summon marauders. Its explained in the changelog but basically recently created marauders will count "double" for the purposes of the construct cap - once they are no longer "recent" (about 60 seconds) they will only count as 1. 

If your cult can only sustain 2 marauders, summoning 1 will lock you out from summoning number two for 60 seconds. If you can sustain 3 then you can't summon the third until at least 60-120 seconds have passed since the second one (that number depends on if the second one was summoned within 60 seconds of the first).

This will encourage cultists to space out their summonings and keep the marauders alive instead of having them go on back-to-back infinite suicide missions. It will also prevent a single cultist from succ'ing the cult dry of power using marauders.

**Comment by Potato-Masher** [comment]
Sounds great. I'm starting to see how annoying marauders are, especially the ones that carry a water tank around so they can smash it and slip EVERYTHING. Dealing with CC is kinda fun until the freaking marauders show up and the chaplain goes missing.

**Comment by Cyberboss** [comment]
Travi!

**Comment by Cyberboss** [comment]
still

**Comment by Cyberboss** [comment]
still

File: code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm [hunk 1]
```diff
@@ -94,10 +94,16 @@
...
+		if(ishuman(L) && L.stat != DEAD)
 			human_servants++
 	construct_limit = human_servants / 4 //1 per 4 human servants, and a maximum of 3 marauders
-	construct_limit = CLAMP(construct_limit, 1, 3)
+	construct_limit = CLAMP(construct_limit - recent_marauders, 1, 3) 
+	if(recent_marauders)
+		to_chat(invoker, "<span class='warning'>The Hierophant Network needs [MARAUDER_SCRIPTURE_SCALING_THRESHOLD / 10] seconds to recover from marauder summoning; recent summoning has limited the number of available marauders by [recent_marauders]!</span>")
+
+
+/datum/clockwork_scripture/create_object/construct/clockwork_marauder/check_special_requirements()
```
Comment (duncathan) [inline code/game/gamemodes/clock_cult/clock_scriptures/scripture_applications.dm]: ???

**Pull request merged**